lib/components/src/bar/button.tsx

```diff
@@ -3,20 +3,20 @@ import { styled, isPropValid } from '@storybook/theming';
 import { transparentize } from 'polished';
 import { auto } from '@popperjs/core';
 
-interface ButtonProps
+interface BarButtonProps
   extends DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
   href?: void;
 }
-interface LinkProps
+interface BarLinkProps
   extends DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement> {
   href: string;
 }
```

Comment:
all interfaces within 1 package MUST all have a unique name for the prebundling to work correctly